ui/admin/app/components/form/worker/index.hbs

```diff
@@ -50,30 +50,6 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <Hds::Form::Fieldset as |F|>
-    <F.Legend>{{t 'resources.worker.form.worker_tags.label'}}</F.Legend>
-    <F.HelperText>{{t 'resources.worker.form.worker_tags.help'}}</F.HelperText>
-    <F.Control>
-      <Hds::Table class='list-wrapper-field config-tags' name='config_tags'>
```

Comment:
should the css associated with config-tags also be removed ?

Reply:
also I wonder if we have related mirage/tests that needs to be cleaned up as well

Reply:
> should the css associated with the config-tags class also be removed ?

Yes it should. Removed.

> also I wonder if we have related mirage/tests that needs to be cleaned up as well

I checked before and I didn't find any tests related to config tags for reading a worker 😶‍🌫️ 
I did find a small test name that needed to be updated though.